src/App.js

```diff
@@ -1,24 +1,74 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from 'react';
+import './AppMain.css';
 
 function App() {
+
+  // Posts: list variable containing Post component objects for every post in the case study, most likely to be imported through a different file
+  //const [Posts, setPosts] = useState([]);
+
+  // dropdownOpen: boolean variable for whether the dropdown menu is open or closed
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+
+  // dropdownHandler: arrow function that switches dropdownOpen value whenever the button is toggled
+  const dropdownHandler = () => {
+    if (dropdownOpen === false) {
+      setDropdownOpen(true);
+    } else {
+      setDropdownOpen(false);
+    }
+  }
+
+  // dropdownArea: arrow function that returns the apporpriate button (and dropdown menu if applicable) depending on whether dropdownOpen is true/false
+  const dropdownArea = () => {
+    if (dropdownOpen === false) {
+      return (
+        <button onClick={dropdownHandler}>
+          <div className="triangle"></div>
+        </button>
+      );
+    } else {
+      return (
+        <button onClick={dropdownHandler}>
+          <div className="triangle up"></div>
+          {/* below is a sample Dropdown Menu placeholder -- this is where the actual component will go */}
+          <div id='dropdownPlaceholder'>dropdown menu</div>
+        </button>
+        
+      );
+    }
+  }
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App'>
+
+      <div className='NavBar'>
+        {/* the TeachLA logo will be a link to ACM TeachLA's website*/ }
+        <a href='https://teachla.uclaacm.com'><img src='assets/TeachLA.PNG' alt='TeachLA icon' id='TeachLA'/></a>
```

Comment:
Instead of linking the image with `src`, I'd import the image as 

```js
import teachLogo from '../assets/....png`;
...
<img src={teachLogo} />
```